tests/integration/queries/test_model_group.py

```diff
@@ -222,6 +222,13 @@ def test_object_detection_metrics(
         assert isinstance(result["bbox"][metric_type], float)
 
 
+def test_object_detection_metrics_bad_id(indico):
+    from indico import IndicoConfig
+    client = IndicoClient()
+    with pytest.raises(IndicoRequestError):
+        _ = client.call(ObjectDetectionMetrics(-1))
```

Comment:
why `_ =` instead of just `client.call(...)`

Reply:
you can do `pytest.raises(Error) as e` and then make assertions on `e` to make sure it has the desired error message

Reply:
mostly because stylistically it is more akin to how that line of code would be called (i.e. returning the object into a variable), obviously (should) never gets there